lib/schemas.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+function Schemas () {
+  this.store = {}
+}
+
+Schemas.prototype.add = function (schema) {
+  const id = schema['$id']
+  delete schema['$id']
```

Comment:
why are you removing `'$id'`?

Reply:
Because in a previous iteration was throwing an error, I'll update this :P